pytroll_collectors/segments.py

```diff
@@ -75,6 +75,39 @@ def __init__(self, config):
         self.logger = logging.getLogger("segment_gatherer")
         self._loop = False
 
+        # Convert check time into int minutes variables
+        for key in self._patterns:
+            if "hour_pattern" in self._patterns[key]:
+                time_conf = self._patterns[key]["hour_pattern"]
+                start_time_str = "00:00"
+                end_time_str = "23:59"
+                delta_time_str = "00:01"
+                if "start_time" in time_conf:
+                    start_time_str = time_conf["start_time"]
+                if "end_time" in time_conf:
+                    end_time_str = time_conf["end_time"]
+                if "delta_time" in time_conf:
+                    delta_time_str = time_conf["delta_time"]
```

Comment:
instead of
```python
start_time_str = "00:00"
if "start_time" in time_conf:
  start_time_str = time_conf["start_time"]
```
you can write:
```python
start_time_str = time_conf.get("start_time", "00:00")
```